subtests/docker_cli/syslog/syslog.py

```diff
@@ -62,16 +66,17 @@ def postprocess(self):
         self.loginfo("Commands: %s" % _command)
         _status = self.stuff['cmdresults'].exit_status
         self.failif(_status, str(self.stuff['cmdresults'].stderr))
-        _check = self.verify_message_logged()
-        self.failif(not _check, "syslog test failed")
+        self.verify_message_logged()
 
     def verify_message_logged(self):
-        with open(self.config['syslogfile']) as f:
-            f.seek(-4096, os.SEEK_END)
-            for line in f:
-                if line.strip().endswith(self.stuff["msg"]):
-                    self.loginfo(line.strip())
-                    return True
```

Comment:
``utils.run('journalctl --flush')`` right before this loop logically negates any possibility of journald/rsyslog buffering having any effect.